siatest/renter.go

```diff
@@ -58,6 +58,15 @@ func (tn *TestNode) DownloadByStream(rf *RemoteFile) (data []byte, err error) {
 	return
 }
 
+// Stream uses the streaming endpoint to download a file.
+func (tn *TestNode) Stream(rf *RemoteFile) (data []byte, err error) {
+	data, err = tn.RenterStreamGet(rf.siaPath)
+	if err == nil && rf.checksum != crypto.HashAll(data) {
```

Comment:
wait, why is `HashAll` being used here? For a `[]byte`, just use `crypto.HashBytes`.